lib/comparison.js

```diff
@@ -277,7 +277,7 @@ helpers.ifEven = function(num, options) {
  */
 
 helpers.ifNth = function(a, b, options) {
-  if (++b % a === 0) {
+  if (b % a === 0) {
```

Comment:
perhaps we should either cast `a` and `b` to numbers and/or do a check to ensure `a` and `b` are numbers before line 280?